src/pages/organize/[orgId]/campaigns/[campId]/surveys/[surveyId]/index.tsx

```diff
@@ -23,7 +23,11 @@ export const getServerSideProps: GetServerSideProps = scaffold(
   },
   {
     authLevelRequired: 2,
-    localeScope: ['layout.organize.surveys', 'pages.organizeSurvey'],
+    localeScope: [
+      'layout.organize.surveys',
+      'pages.organizeSurvey',
+      'layout.organize.survey',
```

Comment:
I see no reason for this line. Is it really required for this to work?

Reply:
how come I missed this! yeah totally agree! thank you. Will fix it right away!